contrib/node/src/python/pants/contrib/node/subsystems/command.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+import os
+from collections import namedtuple
+
+from pants.util.contextutil import get_joined_path
+from pants.util.process_handler import subprocess
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(namedtuple('Command', ['executable', 'args', 'extra_paths'])):
```

Comment:
There's almost nothing node-specific about this class, right?  And it could be generally useful. Maybe add a `# TODO: Consider generalizing this into pants.util.` comment?

Reply:
done.